0.3/schemas/json_schema/collection.schema

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "collection_schema.json",
+    "name": "JSON Schema for NGFF Collection",
+    "description": "Attempt at a JSON Schema to create valid JSON-LD descriptions. Limited to using a few schema.org properties.",
```

Comment:
I know this was used to drive the validation but having a `collection.schema` under the `0.3` namespace feels fairly incorrect. Can we split into a separate branch or potentially a separate directory e.g. `{draft,in-progress}`

Reply:
I'll remove it for now and keep a local copy until we're ready to work on this again.